### PR TITLE
Add I18n support for English

### PR DESCRIPTION
**Pull request by arielo**
Added i18n support for locale strings to help with translating the site.

- Add i18n with language support for English
- Add a global dropdown language switch to the site footer

Related to Code4PuertoRico/suministrospr/issues/55

@jpadilla Are `.mo` files be compiled as part of the deploy process? 

**TODOs:** 

- [X] Updates for tag search translations
- [x] Add language blocks to footer
- [x] Add language blocks for `suministro_list` template text
- [x] Translate footer
- [x] Translate `suministro_list` template text

### CODE REVIEW

**Comment by froi** [comment]
This is freaking awesome. Thanks!!!

**Comment by jpadilla** [comment]
@arielo yeah ideally we can compile those automatically and ignore them from the repo. I think I'd prefer setting up the heroku app to build and run off our Dockerfile instead of fiddling with heroku buildpacks.

I'll start that in a separate PR and let you know.

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,10 @@ ARG PIPENV_ARGS
 ENV LANG C.UTF-8
 ENV PYTHONUNBUFFERED 1
 
+# Add dependencies for i18n
+RUN apt-get update -y
+RUN apt-get install -y gettext
```
Comment (jpadilla) [inline Dockerfile]: I'd suggest tweaking this to improve layer caching:

```
RUN apt-get update && apt-get install -y \
    gettext \
 && rm -rf /var/lib/apt/lists/*
```

**Comment by jpadilla** [comment]
@arielo https://github.com/Code4PuertoRico/suministrospr/pull/133 is now merged. We can switch the stack officially when merging this PR.

Should be able to do `python manage.py compilemessages` here https://github.com/Code4PuertoRico/suministrospr/pull/133/files#diff-86234f17466797048d187a83f8cc04eaR7

**Comment by arielo** [comment]
@jpadilla I rebased and updated the PR. Is anyone willing to translate what is missing?

**Comment by sociovirology** [comment]
> @jpadilla I rebased and updated the PR. Is anyone willing to translate what is missing?

I am willing, but I don't have experience with Django. I've been reading and I think with a little direction I can do it. To start, should I fork https://github.com/arielo/suministrospr/ or https://github.com/Code4PuertoRico/suministrospr?

**Comment by gcollazo** [comment]
@arielo Status?

**Comment by arielo** [comment]
@gcollazo Sorry I hadn't seen this. I needed some help translating/reviewing the main text blocks. it looks it's pretty outdated by now so I'll have to rebase. Maybe I can just duplicate the text in line and we can finish the translation separately and avoid further template conflicts.

**Comment by arielo** [comment]
@gcollazo @sociovirology I rebased this and translated most. Only the main copy needs to be translated here: 
https://github.com/Code4PuertoRico/suministrospr/blob/4912e4df9b0a3153d4c2addc495d11c9c566deb5/suministrospr/suministros/templates/suministros/suministro_list.html#L19

**Comment by froi** [comment]
@gcollazo What's the status of this PR?

Does it need more work or help?

File: suministrospr/suministros/templates/suministros/suministro_list.html [hunk 2]
```diff
@@ -1,32 +1,46 @@
...
 
+{% get_current_language as language_code %}
 <div class="main__about">
   <div class="main__about main__about--paragraph">
     <p>
+     {% if language_code == 'es' %}
       La donación de suministros a las áreas más afectadas del país por el terremoto ha sido un éxito. Esta plataforma, administrada por los mismos donantes y recipientes, busca ser un vehículo de distribución equitativa de estos suministros. De esta manera, la ayuda llega a donde se necesita.
+      {% endif %}
+      {% if language_code == 'en' %}
+      La donación de suministros a las áreas más afectadas del país por el terremoto ha sido un éxito. Esta plataforma, administrada por los mismos donantes y recipientes, busca ser un vehículo de distribución equitativa de estos suministros. De esta manera, la ayuda llega a donde se necesita.
```
Comment (froi) [inline suministrospr/suministros/templates/suministros/suministro_list.html]: Here's one option for this translation

```suggestion
      The donation of supplies to the areas affected by the Puerto Rico earthquakes has been a success. This platform, that is administrated by the same donors and recipients, tries to be a fair distribution method for these supplies. This way, supplies arrive where they are most needed.
```

**Review by froi — CHANGES_REQUESTED** [review]
@arielo Te deje un code suggestion. Una vez eso se applique creo que esto esta good to go.

**Comment by arielo** [comment]
@froi ok it's all yours.

**Review by froi — APPROVED** [review]
Creo que esto esta bien para darle merge. Gracias @arielo 

CC: @gcollazo

**Pull request merged**